code/modules/mob/living/simple_animal/friendly/farm_animals.dm

```diff
@@ -285,6 +304,25 @@
 	..()
 	amount_grown = 0
 
+/mob/living/simple_animal/chick/attack_hand(mob/living/carbon/human/M)
+	. = ..()
+	switch(M.a_intent)
+		if("help")
+			wuv(1,M)
+		if("harm")
+			wuv(-1,M)
+
+/mob/living/simple_animal/chick/proc/wuv(change, mob/M)
+	if(change)
+		if(change > 0)
+			if(M && stat != DEAD)
+				new /obj/effect/temp_visual/heart(loc)
+				emote("me", 1, "moos happily!")
```

Comment:
dunno what kind of chicks you have where you're from, but I don't see them moo often

Reply:
Yeah, whoops.